solr/core/src/java/org/apache/solr/metrics/MetricSuppliers.java

```diff
@@ -154,17 +155,25 @@ public static Clock getClock(PluginInfo info, String param) {
   private static final String EDR_CLAZZ = ExponentiallyDecayingReservoir.class.getName();
   private static final String UNI_CLAZZ = UniformReservoir.class.getName();
   private static final String STW_CLAZZ = SlidingTimeWindowReservoir.class.getName();
+  private static final String STWA_CLAZZ = SlidingTimeWindowArrayReservoir.class.getName();
```

Comment:
I noticed there's no support for the new `SlidingTimeWindowArrayReservoir` class, so I added it here. See [dropwizard docs](https://metrics.dropwizard.io/4.2.0/manual/core.html#sliding-time-window-reservoirs):
> Hint: Try to use our new optimised version of `SlidingTimeWindowReservoir` called `SlidingTimeWindowArrayReservoir`. It brings much lower memory overhead. Also it’s allocation/free patterns are different, so GC overhead is 60x-80x lower then `SlidingTimeWindowReservoir`. Now `SlidingTimeWindowArrayReservoir` is comparable with `ExponentiallyDecayingReservoir` in terms GC overhead and performance. As for required memory, `SlidingTimeWindowArrayReservoir` takes ~128 bits per stored measurement and you can simply calculate required amount of heap.